tools/clang/lib/SPIRV/TypeTranslator.h

```diff
@@ -134,7 +142,7 @@ class TypeTranslator {
 
   /// \brief Translates the given HLSL resource type into its SPIR-V
   /// instructions and returns the <result-id>. Returns 0 on failure.
-  uint32_t translateResourceType(QualType type);
+  uint32_t translateResourceType(QualType type, LayoutRule rule);
 
   /// \brief Returns the alignment and size in bytes for the given type
   /// according to std140.
```

Comment:
Returns the alignment and size in bytes for the given type according to the given LayoutRule.